backend/src/chats/chat/chat.module.ts

```diff
@@ -1,11 +1,17 @@
-import { Module } from '@nestjs/common';
+import { forwardRef, Module } from '@nestjs/common';
 import { ChatService } from './chat.service';
 import { ChatController } from './chat.controller';
 import { TypeOrmModule } from '@nestjs/typeorm';
 import { Chat } from './entities/chat.entity';
+import { AuthModule } from '../../auth/auth.module';
+import { SharedModule } from '../../shared/shared.module';
 
 @Module({
-	imports: [TypeOrmModule.forFeature([Chat])],
+	imports: [
+		TypeOrmModule.forFeature([Chat]),
+		forwardRef(() => AuthModule),
```

Comment:
I don't seem to need to forwardRef here

Reply:
Actually.. It seems to work without the import of the AuthModule and the Shared module.. Maybe those two are only needed when you actually do something withthe auth controller or service. Not just the guard.. (not sure..)

Reply:
Me copy and paste without double checking necessity, never ;)